release-notes/next.yml

```diff
@@ -1,4 +1,12 @@
 releases:
+  43.20250917.1.0:
+    issues:
+      - text: "Rebase onto Fedora 43"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1935"
+      - text: "Migrate existing systems to `iptables-nft` and remove `iptables-legacy`"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1818"
+      - text: "Enable automatic bootloader updates"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1468"
```

Comment:
let's also include https://github.com/coreos/fedora-coreos-tracker/issues/1969 here if you don't mind. I added the `pending-next-release` label to it.

Reply:
I added it :+1: